exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/partition/PruneScanRule.java

```diff
@@ -166,16 +174,28 @@ protected void doOnMatch(RelOptRuleCall call, DrillFilterRel filterRel, DrillPro
     }
 
     if (partitionColumnBitSet.isEmpty()) {
-      logger.debug("No partition columns are projected from the scan..continue.");
+      logger.info("No partition columns are projected from the scan..continue. " +
+          "Total pruning elapsed time: {} ms", totalPruningTime.elapsed(TimeUnit.MILLISECONDS));
```

Comment:
I think this is fine, since we always print a message at the beginning of PruneScanRule, it makes sense to print message indicating when we terminated. Also the message before ("No partition columns...) indicates that we didn't perform actual pruning. My objective was to capture total time spent in PruneScanRule.